src/data/ponder/client/getClients.ts

```diff
@@ -23,7 +23,7 @@ const CLIENT_INFO: Record<number, { url: string; icon: string } | undefined> = {
     url: "https://nounsagora.com",
     icon: "https://nounsagora.com/favicon.ico",
   },
-  11: { url: "https://nouns.farm", icon: "/client/unknown.svg" },
+  11: { url: "https://nouns95.wtf", icon: "/client/unknown.svg" },
```

Comment:
Please add your image to `/public/client` folder, and make this change:
```suggestion
  11: { url: "https://nouns95.wtf", icon: "/client/nouns96.png" },
```